src/Authoring/WinRT.SourceGenerator/AnalyzerReleases.Unshipped.md

```diff
@@ -23,7 +23,6 @@ CsWinRT1015 | Usage | Error | Do not use `DefaultOverloadAttribute` more than on
 CsWinRT1016 | Usage | Error | Exactly one overload should be marked as DefaultOverload
 CsWinRT1017 | Usage | Error | Array types should be one dimensional, not jagged
 CsWinRT1018 | Usage | Error | Array types should be one dimensional
-CsWinRT1019 | Usage | Error | Do not use the `System.Array` type for array parameters or array return values 

```

Comment:
I took this out in a previous PR and missed this at that time. We don't use this rule anymore, we use InvalidType rule now.